### DISCUSSION TITLE
Video Crop example but with actual cropping results

### DISCUSSION THREAD

**Discussion by david-marx** [ValentinH/react-easy-crop#230] [General]
Hi everyone! I tried the cropping with images and videos. Images work great so far, but the video cropping example doesn't show how to get the actual cropping result of the video. Is there an example for the getCroppedImg function, but for videos? Thanks in advance

**Comment by ValentinH** [comment]
Hey 👋
I don't really know if this is possible on the browser. You probably have to send the original video and the cropping coordinates to a backend service where you can use something like `ffmpeg`.
